app/views/govuk_publishing_components/components/_attachment.html.erb

```diff
@@ -83,7 +83,9 @@
       <% end %>
     <% end %>
 
-    <% if attachment.alternative_format_contact_email %>
+    <% if attachment.display_accessible_format_request_form_link? %>
+      <%= link_to "Request an accessible format of this document", "/contact/govuk/request-accessible-format?content_id=#{attachment.content_id}&attachment_id=#{attachment.attachment_id}" %>
```

Comment:
Links require a "govuk-link" class to be accessible and in-line with the Design System.
```suggestion
      <%= link_to "Request an accessible format of this document", "/contact/govuk/request-accessible-format?content_id=#{attachment.content_id}&attachment_id=#{attachment.attachment_id}", class: "govuk-link" %>
```

Reply:
thanks, done 👍